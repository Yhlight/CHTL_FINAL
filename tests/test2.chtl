html {
    head {
        title {
            text { "My CHTL Page" }
        }
    }
    body {
        div {
            id: "main-content";
            class: "container";

            h1 {
                text { "Welcome to CHTL!" }
            }

            # This is a comment that should appear in the HTML
            p {
                text { "This is a paragraph." }
            }
        }
    }
}
