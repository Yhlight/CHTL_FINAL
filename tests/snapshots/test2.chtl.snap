<html>
  <head>
    <title>
      My CHTL Page
    </title>
  </head>
  <body>
    <div id="main-content" class="container">
      <h1>
        Welcome to CHTL!
      </h1>
      <!-- This is a comment that should appear in the HTML -->
      <p>
        This is a paragraph.
      </p>
    </div>
  </body>
</html>
